### DISCUSSION TITLE
Set default app theme

### DISCUSSION THREAD

**Discussion by MattBDev** [dotnet/maui#9231] [Q&A]
I want my app to default to light mode on launch regardless of the OS settings. Is it possible to do this? I don't want my app to have any custom styling either because I like the colors from libraries like WinUI. 
I tried using `Application.Current.UserAppTheme = AppTheme.Light;` but it didn't work. 
Any help is appreciated.

**Comment by xwang-888** [comment]
![image](https://user-images.githubusercontent.com/56998197/187008944-9064bade-34f5-49a0-b280-0705ca69ab63.png)
I think it is effective. Maybe the latest version of Maui fixes your problem.
